core/crypto/crypto.cpp

```diff
@@ -90,6 +125,8 @@ void Crypto::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("verify", "hash_type", "hash", "signature", "key"), &Crypto::verify);
 	ClassDB::bind_method(D_METHOD("encrypt", "key", "plaintext"), &Crypto::encrypt);
 	ClassDB::bind_method(D_METHOD("decrypt", "key", "ciphertext"), &Crypto::decrypt);
+	ClassDB::bind_method(D_METHOD("hmac_digest", "hash_type", "key", "msg"), &Crypto::hmac_digest);
+	ClassDB::bind_method(D_METHOD("constant_time_compare", "hm1", "hm2"), &Crypto::constant_time_compare);
```

Comment:
I was wondering if we should rename the parameters to something more sensible. Would `byte_array`/`other_byte_array` make sense?
There is also the case where the 2 sizes do not match, so a timing attack could still successfully discover the size of the compared value.
Would it make sense to call the first parameter `trusted_bytes` and the second one `received_bytes`, so we can, in the future maybe, improve it knowing which one the user consider trusted (which I think is how `safe_memcmp`s work, but I'm no expert, so the function is fine for now).

Reply:
I renamed the parameters for the function. I didn't do anything about the size comparison shortcut though, as we discussed on IRC.

Reply:
It seems you changed the name of the function parameters, not the one in the `method_bind` (which are the names used by the doc generation tool).
rename `hm1` and `hm2` in `bind_method` to: `trusted`, `received`.
Possibly rename `trusted`, `received` in the method definition to `p_trusted`, `p_received`.

Reply:
And remember to re-run the doctool to update the XML

Reply:
@Faless done. Let me know if you see anything else. Thanks!